app/models/form/lettings/questions/previous_let_type.rb

```diff
@@ -21,4 +21,15 @@ def initialize(id, hsh, page)
     "divider" => { "value" => true },
     "3" => { "value" => "Don’t know" },
   }.freeze
+
+  ANSWER_OPTIONS_2024 = {
```

Comment:
`ANSWER_OPTIONS_FROM_2024` perhaps? just to make clear it's not for this year only and then reverting

Reply:
Is `ANSWER_OPTIONS_AFTER_2024` ok? It would align with the form method